src/pages/guides/nodejs/nitric-and-drizzle.mdx

```diff
@@ -0,0 +1,289 @@
+export const description = 'How to work with Nitric SQL Databases and Drizzle'
+
+# Nitric SQL Databases with Drizzle
+
+This guide demonstrates how to use [Drizzle ORM](https://orm.drizzle.team) with Nitric's [SQL database](/sql) feature. For this guide we will be creating a to-do app. The finished source can be found [here](https://github.com/nitrictech/examples/tree/main/v1/nitric-drizzle)
+
+<Note>
+  SQL databases are currently in preview and only support PostgreSQL deployed to
+  AWS.
+</Note>
+
+## Create a new nitric project
+
+The first step is to create a new Nitric TypeScript project using the [Nitric CLI](/reference/cli).
+
+```bash
+nitric new todo-app ts-starter
+```
+
+## Enabling SQL databases
+
+SQL databases are currently in Preview. To enable this feature in your project add the following to your `nitric.yaml` file
+
+```yaml
+preview:
+  - sql-databases
```

Comment:
Is it `preview-features` or was this changed. If so, there are references to `preview-features` in the websockets documentation.

```suggestion
preview-features:
  - sql-databases
```

Reply:
think it has been preview since v1